src/services/fhirService.ts

```diff
@@ -608,13 +610,18 @@ const fhirService = {
   },
 
   countResources(ownerId: string, params: IParams = {}): Promise<number> {
-    return recordService
-      .searchRecords(ownerId, prepareSearchParameters(params), true)
-      .then(result => result.totalCount);
+    const parameters = prepareSearchParameters({
+      ...params,
+      exclude_tags: [...(params.exclude_tags || []), taggingUtils.generateAppDataFlagTag()],
```

Comment:
We might want to have an array'ed set (or a lodash uniq'ed array) here in case `exclude_tags` already adds the app data rag tag flag bag.